src/GridCell/specAppearance.js

```diff
@@ -0,0 +1,67 @@
+export default [{
+   name: 'backgroundColor',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value',
+   responsive: true
+}, {
+   name: 'backgroundImage',
```

Comment:
Perhaps a specimen that displays the background image related props would be helpful?

Reply:
Im going to do this in another PR if that is ok. I filed an issue. #28